stablehlo/tests/CheckOps.td

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef STABLEHLO_DIALECT_CHECK_OPS
+#define STABLEHLO_DIALECT_CHECK_OPS
+
+include "stablehlo/dialect/Base.td"
+
+//===----------------------------------------------------------------------===//
+// CHECK dialect.
+//===----------------------------------------------------------------------===//
+
+def CHECK_Dialect : Dialect {
+  let name = "check";
+  let cppNamespace = "::mlir::stablehlo::check";
+
+  let description = [{
+    Opset to implement test assertions for StableHLO interpreter.
```

Comment:
"to implement checks for testing the StableHLO interpreter"? Since we're moving away from the terminology involving assertions.